source/common/upstream/health_checker_base_impl.h

```diff
@@ -123,6 +124,8 @@ class HealthCheckerImplBase : public HealthChecker,
   void incHealthy();
   void incDegraded();
   std::chrono::milliseconds interval(HealthState state, HealthTransition changed_state) const;
+  std::chrono::milliseconds
+  intervalWithJitter(uint64_t base_time_ms, const std::chrono::milliseconds interval_jitter) const;
```

Comment:
Either make `interval_jitter` const ref or drop the `const` qualifier (see https://clang.llvm.org/extra/clang-tidy/checks/readability-avoid-const-params-in-decls.html)

Reply:
ok